### PR TITLE
Added loader state for all requests [WD-1032]

### PR DESCRIPTION
**Pull request by lorumic**

### CODE REVIEW

**Review by edlerd — COMMENTED** [review]
Add loading indicator to:
* instance terminal tab
* instance creation > select image modal / table

Needs fix:
* projects > custom create profile is stuck in loading state

File: static/sass/styles.scss [hunk 1]
```diff
@@ -103,3 +103,7 @@
 .no-grid-gap {
   grid-gap: 0;
 }
+
+.u-loader {
+  margin: 0.5rem;
```
[the remark this answers is not in the document]
Comment (lorumic) [reply static/sass/styles.scss:108]: `auto` doesn't work well in side panels unfortunately:

image

Should I set it as `margin: $sp-large;` instead?

image

**Comment by lorumic** [comment]
> Add loading indicator to:
> 
> * instance terminal tab
> * instance creation > select image modal / table
> 
> Needs fix:
> 
> * projects > custom create profile is stuck in loading state

Done @ [b18f923](https://github.com/canonical/lxd-ui/commit/b18f9233a9b057389391606e41a82d8918aa84a4)

**Review by edlerd — APPROVED** [review]

File: static/js/modals/ImageSelect.tsx [hunk 2]
```diff
@@ -69,16 +69,8 @@ const ImageSelect: FC<Props> = ({ onClose, onSelect }) => {
...
-  if (isLciLoading) {
-    return <Loader text="Loading Linux Containers images..." />;
-  }
-
-  if (isCiLoading) {
-    return <Loader text="Loading Canonical images..." />;
-  }
-
-  const images = canonicalImages.concat(linuxContainerImages);
+  const isLoading = isCiLoading || isLciLoading;
```
Comment (edlerd) [inline static/js/modals/ImageSelect.tsx:72]: @lorumic made a little adjustment here, to open the modal immediately and show the loading spinner in the table.

**Pull request merged**